drivers/stm32/README.md

```diff
@@ -1,51 +1,6 @@
 # Libcanard Driver for STM32 Microcontrollers
 
-This is a compact and simple driver for STM32 microcontrollers.
-It has no dependencies besides a tiny part of the standard C library,
-and can be used with virtually any operating system or on bare metal
-(so far it has been tested at least with ChibiOS).
+This is a heavily modified version of the bundled STM32 driver. It uses RX/TX irq driven transmissions.
```

Comment:
With this changeset, libcanard as a whole has become tightly with both mordor and STM32. This is fine for our purposes, but kind of goes against the original design of the library and precludes us from ever being able to push back upstream.

Does it make sense for us to kill this as a submodule and pull it into mordor directly? I think the license is permissive enough to allow that, and I think it will make the code easier to follow and bisect.

Reply:
Mostly agree, but doing so might make it slightly harder to pull from upstream, no?

Reply:
Yes, but it will already be difficult to pull from upstream given the magnitude of these changes, even if kept as a fork. Conflicts are likely when we've diverged this far....

Reply:
@robweiss Any objections to this? I think @wesen is also onboard with this.

Reply:
upstream is ... at best.